docker/puppetdb/docker-entrypoint.d/30-configure-ssl.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ ! -f "/etc/puppetlabs/puppet/ssl/certs/${HOSTNAME}.pem" ] && [ "$USE_PUPPETSERVER" = true ]; then
   set -e
-  /ssl.sh
+  DNS_ALT_NAMES="${DNS_ALT_NAMES}" /ssl.sh "$HOSTNAME"
```

Comment:
While it's not necessary for us to specify this values here, since ssl.sh will find these ENV variables itself, I wanted to make it clear where they're being used in this container definition since ssl.sh doesn't live in this repo.